web/jsp/comment.jsp

```diff
@@ -0,0 +1,51 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Vartotojas
+  Date: 12-Mar-19
+  Time: 11:38
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
It would be better to delete such information

Reply:
deleted